### PR TITLE
 Document pre-baked image

### PR DESCRIPTION
**Pull request by jpechane**
Apply after https://github.com/debezium/microservices-lab/pull/18

### CODE REVIEW

File: vm-setup/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Debezium Microservices Lab Vagrant image
+
+This is a source repository for [Vagrant image](https://app.vagrantup.com/debezium/boxes/microservices-lab) serving as a Debezium microservices lab environment.
+
+## Image usage
+There are two ways how to use the image
+* (recommended) Use provided `Vagrant` file. In this case just downlod the `Vagrantfile` into an arbitrary directory and issue `vagrant up` command.
```
Comment (gunnarmorling) [inline vm-setup/README.md]:
```suggestion
* (recommended) Use provided `Vagrant` file. In this case just download the `Vagrantfile` into an arbitrary directory and issue `vagrant up` command.
```

File: vm-setup/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+After a while the new virtual machine will be spawned and you could log to it via `vagrant ssh`.
+* (advanced) Create the envrionment form scratch using `vagrant init debezium/microservices-lab` command.
```
Comment (gunnarmorling) [inline vm-setup/README.md]:
```suggestion
* (advanced) Create the environment from scratch using `vagrant init debezium/microservices-lab` command.
```

File: vm-setup/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+In this case the user is responsible for configuration of networking, port-forwarding and capacity allocation.
+
+## Building the image
+To build the image from scratch it is necessary to folow this procedure
```
Comment (gunnarmorling) [inline vm-setup/README.md]:
```suggestion
To build the image from scratch it is necessary to follow this procedure
```

File: vm-setup/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+* Use `Vagrantfile.build`as the Vagrant configuration file.
+* Start the virtual machine using `vagrant up`.
+* Log into virtual machine using `vagrant ssh`.
+* As `root` execute
+```
+$ dnf clean all
+
+# It is expected that the following command will fill the disk and generate an error
```
Comment (gunnarmorling) [inline vm-setup/README.md]: What's the purpose of this?

File: vm-setup/Vagrantfile [hunk 5]
```diff
@@ -23,7 +23,6 @@ Vagrant.configure("2") do |config|
   # within the machine from a port on the host machine. In the example below,
   # accessing "localhost:8080" will access port 80 on the guest machine.
   config.vm.network "forwarded_port", guest: 8443, host: 8443
-  config.vm.network "forwarded_port", guest: 80, host: 80
```
Comment (gunnarmorling) [inline vm-setup/Vagrantfile]: Why is this not needed any more?

File: vm-setup/Vagrantfile.init [hunk 6]
```diff
@@ -0,0 +1,8 @@
+Vagrant.configure("2") do |config|
+  config.vm.synced_folder ".", "/vagrant", type: "rsync"
+  config.vm.provider "virtualbox" do |vb|
+   vb.memory = "4096"
```
Comment (gunnarmorling) [inline vm-setup/Vagrantfile.init]: Ok, so are we sure that 4G is enough? I.e. the CPU count of 2 is the deciding factor to make things stable?

File: vm-setup/Vagrantfile (hunk 5 above)
Comment (jpechane) [reply vm-setup/Vagrantfile]: It never worked, port 80 is a privileged port so unl;ess vb was running as a root it port forwarding was not enabled.

File: vm-setup/README.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+## Image usage
+There are two ways how to use the image
+* (recommended) Use provided `Vagrant` file. In this case just download the `Vagrantfile` into an arbitrary directory and issue `vagrant up` command.
+After a while the new virtual machine will be spawned and you could log to it via `vagrant ssh`.
+* (advanced) Create the environment from scratch using `vagrant init debezium/microservices-lab` command.
+In this case the user is responsible for configuration of networking, port-forwarding and capacity allocation.
+
+## Building the image
+To build the image from scratch it is necessary to follow this procedure
+* Use `Vagrantfile.build`as the Vagrant configuration file.
```
Comment (gunnarmorling) [inline vm-setup/README.md]: How do I do this exactly?

File: vm-setup/README.md (hunk 4 above)
Comment (jpechane) [reply vm-setup/README.md]: It is for better compressibility of the disk image - recommendation from vagrant

File: vm-setup/Vagrantfile.init (hunk 6 above)
Comment (jpechane) [reply vm-setup/Vagrantfile.init]: No we are not, that's why I kept 6 GB in the default Vagrant file

File: vm-setup/README.md (hunk 7 above)
Comment (jpechane) [reply vm-setup/README.md]: Unfortunately only copy /rename - I've not found any other way

File: vm-setup/Vagrantfile.build [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+  # Disable automatic box update checking. If you disable this, then
+  # boxes will only be checked for updates when the user runs
+  # `vagrant box outdated`. This is not recommended.
+  # config.vm.box_check_update = false
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine. In the example below,
+  # accessing "localhost:8080" will access port 80 on the guest machine.
+  config.vm.network "forwarded_port", guest: 8443, host: 8443
+  config.vm.network "forwarded_port", guest: 80, host: 80
```
Comment (gunnarmorling) [inline vm-setup/Vagrantfile.build]:
```suggestion
```

**Comment by gunnarmorling** [comment]
I'll get to the application update PR next and then do a complete test run, hopefully identifying any missing images.

**Pull request closed without merging**